apps/todoapp/src/app/todos/services/discord.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from '@angular/core';
+import { DiscordUser } from '../types/discordUser.interface';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DiscordService {
+  async getUserAvatar(discordUser: DiscordUser): Promise<string> {
+    return `https://cdn.discordapp.com/avatars/${discordUser.discordId}/${discordUser.avatar}.png`;
```

Comment:
if a user does not have an avatar, the discordUser.avatar = null so this won't work, a default image should be loaded in this case

Reply:
i reworked it so it is getting default avatar when no avatar is present

Reply:
we need to check together, it does not work on my side...